radio/src/targets/taranis/CMakeLists.txt

```diff
@@ -190,6 +190,32 @@ elseif(PCB STREQUAL X7)
     add_definitions(-DRADIO_TX12)
     add_definitions(-DEEPROM_VARIANT=0x4002)
     add_definitions(-DMANUFACTURER_RADIOMASTER)
+  elseif(PCBREV STREQUAL TX12MK2)
+    set(DEFAULT_INTERNAL_MODULE CROSSFIRE CACHE STRING "Default internal module")
+    set(INTERNAL_MODULE_SERIAL YES)
+    set(FLAVOUR tx12mk2)
+    set(NAVIGATION_TYPE x7)
+    set(CPU_TYPE STM32F4)
+    set(CPU_TYPE_FULL STM32F407xE)  # for size report
+    add_definitions(-DSTM32F407xx)
+    set(LINKER_SCRIPT targets/taranis/stm32f4_flash.ld)
+    set(FIRMWARE_TARGET_SRC
+            ${FIRMWARE_TARGET_SRC}
+            startup_stm32f40_41xxx.s
+            ../../thirdparty/STM32F4xx_DSP_StdPeriph_Lib_V1.8.0/Libraries/STM32F4xx_StdPeriph_Driver/src/stm32f4xx_flash.c
+            )
+    add_definitions(-DSTM32F40_41xxx -DSTM32F407xx)
+    set(ROTARY_ENCODER YES)
+    set(LUA_EXPORT lua_export_tx12mk2)
+    set(BLUETOOTH NO)
+    set(USB_CHARGER YES)
+    set(AUX_SERIAL ON)
+    add_definitions(-DRADIO_TX12MK2)
+    add_definitions(-DEEPROM_VARIANT=0x4008)
```

Comment:
I think this line gets the chop for us?

Reply:
That would be a question for the Companion import side, unsure where it currently stand

Reply:
I believe the policy was going to be zero compatibility (implied) for new radios... but if it's a minor thing to include I'm all for it as it would be a pleasant surprise when it actually works ;)

Reply:
I think it is going to be delivered with EdgeTX on it, right?
If so, no need for compatibility with OpenTX